packages/connect/src/device/__tests__/checkFirmwareRevision.test.ts

```diff
@@ -0,0 +1,104 @@
+import { DeviceModelInternal } from '@trezor/protobuf';
+import { checkFirmwareRevision, CheckFirmwareRevisionParams } from '../checkFirmwareRevision';
+import { FirmwareRelease, FirmwareRevisionCheckResult } from '../../exports';
+import * as utilsAssets from '../../utils/assets';
+
+const ONLINE_RELEASES_JSON_MOCK: FirmwareRelease[] = [
+    {
+        required: false,
+        version: [1, 12, 1],
+        bootloader_version: [1, 12, 1],
+        min_firmware_version: [1, 12, 0],
+        min_bootloader_version: [1, 12, 0],
+        url: 'firmware/t1b1/trezor-t1b1-1.12.1.bin',
+        url_bitcoinonly: 'firmware/t1b1/trezor-t1b1-1.12.1-bitcoinonly.bin',
+        fingerprint: '3c694191f5b66a65cb5bb209adbf113cb40209e644b77162ba996bb7ee8f382b',
+        fingerprint_bitcoinonly: '985fb6a8c87f7547fb810f6c4a8331ebf19c677445810358778eb21eca78a181',
+        firmware_revision: '1eb0eb9d91b092e571aac63db4ebff2a07fd8a1f',
+        changelog:
```

Comment:
nitpick: since this i only a mock, you dont need to put real changelog here

Reply:
https://github.com/trezor/trezor-suite/pull/13495/commits/7b2d5ae513b9c6c2be44bfeb57e4646ef54f8ae2